$NetBSD$
.. work/gr-fcdproplus/lib/hid/hid.c:47:20: fatal error: libusb.h: No such file or directory

--- CMakeLists.txt.orig	2014-10-03 09:19:29.000000000 +0900
+++ CMakeLists.txt	2014-10-03 09:19:47.000000000 +0900
@@ -44,7 +44,7 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_SO
 ########################################################################
 if(CMAKE_COMPILER_IS_GNUCXX AND NOT WIN32)
     #http://gcc.gnu.org/wiki/Visibility
-    add_definitions(-fvisibility=hidden)
+    add_definitions(-fvisibility=hidden -I@@PREFIX@@/include/libusb-1.0)
 endif()
 ########################################################################
 # Find boost
